src/Compiler/CompilerOptions.cpp

```diff
@@ -285,6 +285,11 @@ llvm::cl::opt<std::string> reportHeapAfter("report-heap-after",
                    "<output-files-base-path>.heap.log"),
     llvm::cl::init(""), llvm::cl::cat(OnnxMlirOptions));
 
+llvm::cl::opt<bool> funcCallErrorExit("func-call-error-exit",
+    llvm::cl::desc("Execution failed when external function call failed."
+                   " Currently only zDNN calls in NNPA are supported."),
+    llvm::cl::init(false), llvm::cl::cat(OnnxMlirOptions));
+
```

Comment:
Move this inside the NNPA folder, since it is for NNPA only. Perhaps, prefixing NNPA options with `nnpa` is easier for user.